state/user_test.go

```diff
@@ -229,16 +229,29 @@ func (s *UserSuite) TestRemoveUser(c *gc.C) {
 func (s *UserSuite) TestDisable(c *gc.C) {
 	user := s.Factory.MakeUser(c, &factory.UserParams{Password: "a-password"})
 	c.Assert(user.IsDisabled(), jc.IsFalse)
+	c.Assert(s.activeUsers(c), jc.DeepEquals, []string{"test-admin", user.Name()})
 
 	err := user.Disable()
 	c.Assert(err, jc.ErrorIsNil)
 	c.Assert(user.IsDisabled(), jc.IsTrue)
 	c.Assert(user.PasswordValid("a-password"), jc.IsFalse)
+	c.Assert(s.activeUsers(c), jc.DeepEquals, []string{"test-admin"})
```

Comment:
It's interesting that we are expecting disabled user to be listed as active... Are you sure?